lib/src/rules/always_put_required_named_parameters_first.dart

```diff
@@ -12,26 +12,26 @@ const _desc = r'Put required named parameters first.';
 const _details = r'''
 **DO** specify `required` on named parameter before other named parameters.
 
-**GOOD:**
-```dart
-m({required a, b, c}) ;
-```
-
 **BAD:**
 ```dart
 m({b, c, required a}) ;
 ```
 
 **GOOD:**
 ```dart
-m({@required a, b, c}) ;
+m({required a, b, c}) ;
 ```
 
 **BAD:**
 ```dart
 m({b, c, @required a}) ;
 ```
 
+**GOOD:**
+```dart
+m({@required a, b, c}) ;
```

Comment:
Shouldn’t we use `required` keyword instead of the deprecated annotation?

Reply:
The documentation shows `good` and `bad` for both the `required` and `@required`. I guess it is there because it still can used.